app/scripts/metamask-controller.js

```diff
@@ -475,6 +475,10 @@ export default class MetamaskController extends EventEmitter {
     });
 
     this.phishingController = new PhishingController();
+    if (process.env.IN_TEST) {
+      this.phishingController.updatePhishingLists();
```

Comment:
Does this line need to be in the `IN_TEST` block? I thought it made sense to update the phishing lists right away. That's what the old implementation did as well ([see here](https://github.com/MetaMask/controllers/blob/62ba2a3e8d7959b6dcbc0cf4981209dc21bc04b4/src/third-party/PhishingController.ts#L136))